packages/angular/build/src/builders/application/build-action.ts

```diff
@@ -369,6 +375,21 @@ function* emitOutputResults(
   // If there are template updates and the incremental update was background only, a component
   // update is possible.
   if (hasTemplateUpdates && incrementalResult.background) {
+    // Template changes may be accompanied by stylesheet changes and these should also be updated hot when possible.
+    if (hasCssUpdates) {
```

Comment:
NIT: Grouping the updated map files with this incremental update will make the debugging process less awkward.